File: 2.Algorithm/99.ETC/숫자 표현/숫자 표현.java
class Solution {
    public int solution(int n) {
        int answer = 1;
        for (int i = 1; i < n/2+1; i++){
            int k = 0;
            int j = 0;
            while (k < n){
                k += (i+j);
                j += 1;
                if (k == n){
                    answer++ ;
                    break;
                }
            }
        }
        return answer;
    }
}

File: 2.Algorithm/99.ETC/숫자 표현/숫자 표현.py
def solution(n):
    answer = 1
    for i in range(1,int(n/2)+1):
        k,j = 0,0
        while k < n:
            k += (i+j)
            j+=1
            if k == n:
                answer +=1
                break
    return answer